### PR TITLE
DOCS-4394 Adds Error Tracking for Logs Documentation

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Adds an Error Tracking for Logs section to the Logs docs.

### Motivation

DOCS-4394 

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/logs/error_tracking/explorer.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    tag: 'Blog'
+    text: 'Make sense of application issues with Datadog Error Tracking'
+  - link: '/logs/error_tracking/'
+    tag: 'Documentation'
+    text: 'Learn about Error Tracking for Logs'
+---
+
+## Overview
+
+The [Error Tracking for Logs Explorer][1] intelligently groups error logs into issues that help you understand and triage the root cause of errors by cutting through the noise and reducing thousands to millions of raw data points to a couple of issues consisting of diagnostic data such as stack traces, error distributions, and code snippets.
```
Comment (kayayarai) [inline content/en/logs/error_tracking/explorer.md:17]:
```suggestion
The [Error Tracking for Logs Explorer][1] intelligently groups error logs into issues that help you understand and triage the root cause of errors by cutting through the noise and reducing thousands or millions of raw data points to a couple of issues consisting of diagnostic data such as stack traces, error distributions, and code snippets.
```

File: content/en/logs/error_tracking/explorer.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+### Time range
+
+{{< img src="real_user_monitoring/error_tracking/time_range.png" alt="Error Tracking Time Range" style="width:70%" >}}
+
+The time range appears on the top right of the Error Tracking Explorer as a timeline. This feature allows you to display issues having error occurrences within the selected time period. Change the time range by selecting a preset range from the dropdown.
+
+### Facets
+
+Error Tracking automatically indexes a predefined list of attributes from your issues and creates facets out of it. A facet displays all the distinct members of an attribute for the selected time period and provides some basic analytics, such as the number of issues represented. Facets allow you to pivot or filter your issues based on the given attribute.
```
Comment (kayayarai) [inline content/en/logs/error_tracking/explorer.md:36]:
```suggestion
Error Tracking automatically indexes a predefined list of attributes from your issues and creates facets for them. Clicking a facet filters the data to all available distinct values of the corresponding attribute, for the selected time period. It also provides some basic analytics, such as the number of issues represented. Facets allow you to pivot or filter your issues based on the given attribute.
```

File: content/en/logs/error_tracking/explorer.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Inspect an issue
+
+Click on any issue to open the issue panel which contains information about seasonality patterns, stack traces, and the error's distribution across environments and sources. The **What Happened** section displays the first and last versions are impacted with timestamps. This metadata exists beyond your standard log retention.
```
Comment (kayayarai) [inline content/en/logs/error_tracking/explorer.md:40]:
```suggestion
Click on any issue to open the issue panel which contains information about seasonality patterns, stack traces, and the error's distribution across environments and sources. The **What Happened** section displays the first and last versions that are impacted by the error, plus an indication of when it they were impacted. This metadata is kept beyond your standard log retention.
```

File: content/en/logs/error_tracking/browser_and_mobile.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+    tag: 'Blog'
+    text: 'Make sense of application issues with Datadog Error Tracking'
+  - link: '/logs/error_tracking/explorer/'
+    tag: 'Documentation'
+    text: 'Learn about the Error Tracking Explorer'
+---
+
+## Overview
+
+Error Tracking processes errors collected from browser and mobile Datadog SDKs. Whenever an error containing a stack trace is collected, Error Tracking processes and groups it under an issue, or group of similar errors.
```
Comment (kayayarai) [inline content/en/logs/error_tracking/browser_and_mobile.md:17]:
```suggestion
Error Tracking processes errors collected from browser and mobile Datadog SDKs. Whenever an error containing a stack trace is collected, Error Tracking processes and groups it under an _issue_, which is a grouping of similar errors. 
```

**Review by kayayarai — COMMENTED** [review]
Just a few tiny changes, will commit them myself

File: content/en/logs/error_tracking/_index.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+{{< beta-callout btn_hidden="true" >}}
+  Error Tracking for Logs is in beta.
+{{< /beta-callout >}} 
+
+## Overview
+
+It is critical for your system’s health to consistently monitor the errors collected by Datadog. When there are many individual error events, it becomes hard to prioritize errors for troubleshooting. 
+
+Once you have setup logs for **Browser and Mobile** or **Backend** error tracking, the issue list populates with cards.  You can sort by volume or age, and filter by all custom and default facets on your logs.
```
Comment (kayayarai) [inline content/en/logs/error_tracking/_index.md:23]:
```suggestion
Once you have setup logs for **Browser and Mobile** or **Backend** error tracking, the issue list populates with cards. You can sort by volume or age, and filter by all custom and default facets on your logs.
```

**Review by kayayarai — APPROVED** [review]
Ready to 🚀  when the checks have passed.

**Review by AyushK1 — APPROVED** [review]

**Pull request merged**